app/models/job_io_wrapper.rb

```diff
@@ -63,7 +63,7 @@ def size
     nil # unable to determine
   end
 
-  def file_set(use_valkyrie: false)
+  def file_set(use_valkyrie: Hyrax.config.query_index_from_valkyrie)
```

Comment:
This allows setting `use_valkyrie` from a single point.  We may want to consider this for other places that have a `use_valkyrie` parameter.